README.md

```diff
@@ -1,3 +1,123 @@
 # UMBRELLIO guidelines
 
-In active development
+This manual describes the style and minimal requirements of maintaining and supporting the UMBRELLIO's github repositories.
+
+## Table of contents:
+
+* [Github Repository Design](#github-repository-design)
+  * Own projects
```

Comment:
Why is there no such paragraph, when there is for "Forks"?

Reply:
Yep, useless section.